apis/v1beta1/opentelemetrycollector_types.go

```diff
@@ -110,6 +113,7 @@ type OpenTelemetryCollectorSpec struct {
 	// functionality is only available if one of the valid modes is set.
 	// Valid modes are: deployment, daemonset and statefulset.
 	// +optional
+	// +kubebuilder:validation:Enum=deployment;daemonSet;statefulSet;
```

Comment:
this is not correct

Reply:
This is a mistake. I'll remove the extra  `;`.